OracleGoldenGate/21c/install-deployment.sh

```diff
@@ -44,6 +44,16 @@ function ogg_installer_setup() {
     unzip -q "${INSTALLER}" -d "/tmp/installer" || abort "Unzip operation failed for '${INSTALLER}'"
     chmod -R o=g-w             "/tmp/installer"
 }
+##
+##
+## o g g _ i n s t a l l _ o p t i o n
+##
+function ogg_install_option() {
+    local fastcopy_file
+    fastcopy_file=$(find /tmp/installer -name fastcopy.xml -print -quit)
+    [[ -z "${fastcopy_file}" ]] && abort "The file 'fastcopy.xml' could not be located."
+    perl -ne '/^.* INSTALL_TYPE="(.*)" .*$/ && print $1' "${fastcopy_file}"
```

Comment:
Perl is not used in this project.

Reply:
Thank you for the information. I have modified this to use `awk`